cadc-gms/src/main/java/org/opencadc/auth/StandardIdentityManager.java

```diff
@@ -307,7 +306,7 @@ public String toDisplayString(Subject subject) {
         return null;
     }
 
-    private void validateOidcAccessToken(Subject s) {
```

Comment:
This method rename is unnecessary/misleading so should be reverted: the original name is correct.

Reply:
Will fix.